widget/scroller_test.go

```diff
@@ -43,6 +43,25 @@ func TestScrollContainer_MinSize(t *testing.T) {
 	assert.Equal(t, 0, scroll.Offset.Y)
 }
 
+func TestScrollContainer_ScrollToTop(t *testing.T) {
+	rect := canvas.NewRectangle(color.Black)
+	rect.SetMinSize(fyne.NewSize(500, 50))
+	scroll := NewScrollContainer(rect)
+	scroll.ScrollToTop(true)
+	Y := scroll.Offset.Y
+	assert.Equal(t, 0, Y)
+}
+
+func TestScrollContainer_ScrollToBottom(t *testing.T) {
+	rect := canvas.NewRectangle(color.Black)
+	rect.SetMinSize(fyne.NewSize(500, 50))
+	scroll := NewScrollContainer(rect)
+	scroll.ScrollToBottom(true)
+	Y := 50 + scroll.MinSize().Height
+	ExpectedY := scroll.Content.Size().Height - scroll.Size().Height + scroll.MinSize().Height
```

Comment:
I think that here `Y` is the expected value and `ExpectedY` is the actual value. If that's correct can you please update naming and parameter order below?

Reply:
Done.